src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -7,11 +7,10 @@
     <Configurations>Debug;Release;Debug + MobileApps</Configurations>
   </PropertyGroup>
 
-  <!-- Mobile and legacy targets are disabled by default to speed up dev box -->
+  <!-- Mobile targets are disabled by default to speed up dev box -->
   <PropertyGroup Condition="'$(INCLUDE_MOBILE_AND_LEGACY_TFM)' != ''">
```

Comment:
```suggestion
  <PropertyGroup Condition="'$(INCLUDE_MOBILE_TFM)' != ''">
```

Reply:
It's going to be tricky to update the builds at the same time, since this variable is not part of the yaml, but in the build settings. Let's do a build var cleanup after this gets merged.